.github/workflows/ci.yml

```diff
@@ -0,0 +1,193 @@
+name: CI
+
+on: [pull_request, push]
+
+jobs:
+  lints:
+    name: Rustfmt and clippy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install packages
```

Comment:
Hmm, why can't this be folded into a `packages: libasound2-dev libudev-dev` line?